manifests/install.pp

```diff
@@ -19,8 +19,13 @@
     }
     ensure_packages([$_running_kernel_devel_package], { 'before' => Package['falco'] })
 
-    $_package_deps = ['dkms', 'make']
-    ensure_packages($_package_deps, { 'before' => Package['falco'] })
+    if $falco::driver == 'kmod' {
+      $_package_deps = ['dkms', 'make']
```

Comment:
I think we should not do this. The packages dkms, make, llvm and clang are quite generic and could be provided by other profiles already. Please add a parameter like `$manage_dependencies` or `$manage_llvm`.. to make the installation optional.

Reply:
I added a manage_dependencies parameter, I hope that's better?